doc/configuration.md

```diff
@@ -3,14 +3,14 @@
 The Ignition configuration is a JSON document conforming to the following specification, with **_italicized_** entries being optional:
 
 * **ignition** (object): metadata about the configuration itself.
-  * **version** (string): the semantic version number of the spec. Must be at least `2.0.0`.
```

Comment:
Can this page have a link to the last Ignition v1 docs? Or do it right before v2.